### ISSUE TITLE
💅 useSingleCaseStatement shouldn't report break statements

### ISSUE THREAD

**Issue by flexagoon** [biomejs/biome#2696]
### Environment information

```bash
CLI:
  Version:                      1.7.2
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_LOG_DIR:                unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v21.7.3"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         "npm/10.5.0"

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 true

Linter:
  Recommended:                  false
  All:                          true
  Rules:                        performance/noDelete = "off"
                                style/noDefaultExport = "off"
                                style/noNamespaceImport = "off"
                                style/useNamingConvention = "off"

Workspace:
  Open Documents:               0
```

### Rule name

lint/style/useSingleCaseStatement

### Playground link

https://biomejs.dev/playground/?lintRules=all&code=YwBvAG4AcwB0ACAAZgBvAG8AIAA9ACAAJwBiAGEAcgAnADsACgBzAHcAaQB0AGMAaAAgACgAZgBvAG8AKQAgAHsACgAgACAAYwBhAHMAZQAgACcAYgBhAHoAJwA6AAoAIAAgAGMAYQBzAGUAIAAnAGIAYQByACcAOgAKACAAIAAgACAAYwBvAG4AcwBvAGwAZQAuAGwAbwBnACgAMQApADsACgAgACAAIAAgAGIAcgBlAGEAawA7AAoAIAAgAGQAZQBmAGEAdQBsAHQAOgAKACAAIAAgACAAYwBvAG4AcwBvAGwAZQAuAGwAbwBnACgAMgApADsACgB9AA%3D%3D

### Expected result

The rule should not be triggered when a switch case only contains a single statement and a `break;`, since that is how you generally use switch statements and wrapping every single statement in `{}` is inconvenient.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by dyc3** [comment]
I'd like to pick this up.
